test/per_file_coverage.sh

```diff
@@ -60,7 +60,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/watchdog:85.7" # Death tests within extensions
 "source/extensions/watchdog/profile_action:85.7"
 "source/server:94.4" # flaky: be careful adjusting. See https://github.com/envoyproxy/envoy/issues/15239
-"source/server/admin:95.8"
+"source/server/admin:95.7"
```

Comment:
drive-by comment: rather than doing this can you add more tests to improve coverage? Do you know what changed to cause this to be needed?

Reply:
this is because I deleted the unneeded prometheus namespaces API which results in the decrease of the number of lines in server/admin.. all the code path I added is already covered.